test/e2e/simctl-e2e-specs.js

```diff
@@ -270,5 +272,30 @@ describe('simctl', function () {
         fullList.runtimes.filter((el) => el.identifier.includes('iOS')).length.should.be.above(0);
       });
     });
+
+    describe('getScreenshot', function() {
+      it('should get a base64 string', async function () {
+        const image = await simctl.getScreenshot();
+
+        // Use a rough regex to check if a returned value is base64 string
+        const regex = new RegExp(/[A-Za-z0-9+/=]/);
+        expect(regex.test(image)).to.be.true;
+      });
+    });
+
+    describe('pushNotification', function() {
+      it('should not throw an error when sending a push notification', function () {
+        const payload = {
+          'Simulator Target Bundle': 'com.apple.Preferences',
+          'aps': {
+            'alert': 'This is a simulated notification!',
+            'badge': 3,
+            'sound': 'default'
+          }
+        };
+
+        expect(async () => (await simctl.pushNotification(payload))).to.not.throw();
```

Comment:
for promises you could also use `should.be.fulfilled`

Reply:
thanks, addressed as https://github.com/appium/node-simctl/pull/175/commits/167e91e6a96f8a569dee78938f877b2955c76272